docs/boshdirector.adoc

```diff
@@ -28,114 +40,307 @@ Click on the zone; you'll notice an NS-type entry for the subdomain, as shown be
+
+[.thumb]
+.Ops Manager Dashboard
+image::orange-director-gcp.png[Ops Manager Dashboard]
+
+==== Google Cloud Platform Config
 
-Select the "internal authentication" method when setting up access to your Ops Manager.  Select a username, password, passphrase.
+. In Ops Manager, click on the `Google Cloud Platform` tile (which should be orange).  This will take you to the `Google Config` section of the tile.
 
-=== GCP Config
++
+[.thumb]
+.Google Cloud Platform
+image::gcp-config.png[Google Cloud Platform]
 
-On http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#gcp-config[this step^], just specify your project id ({project_id}) and press Save.
+. As per the http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#gcp-config[documentation^], enter the following values for the specified fields, and accept defaults for all other fields.  Then click `Save`.
 
-NOTE:  if you encounter an error message stating that your ops manager service account cannot see your GCP project, it is an indication of a possible bug whereby the service account was created with insufficient permissions.  Resolve this issue by navigating in your Google Cloud Console to the IAM & Admin -> IAM section.  Configure the service account named _{env_name} Ops Manager VM Service Account_ by adding the role _Compute Security Admin_ (don't foget to press "Save").
++
+NOTE:  If you encounter an error message stating that your ops manager service account cannot see your GCP project, it is an indication of a possible bug whereby the service account was created with insufficient permissions.  Resolve this issue by navigating in your Google Cloud Console to the IAM & Admin -> IAM section.  Configure the service account named _{env_name} Ops Manager VM Service Account_ by adding the role _Compute Security Admin_ (don't foget to press "Save").  If you don't see the _{env_name} Ops Manager VM Service Account_ service account in the IAM section, you can add the required permissions by executing the following command: `gcloud projects add-iam-policy-binding {project_id} --member 'serviceAccount:{env_name}-opsman@{project_id}.iam.gserviceaccount.com' --role 'roles/editor'`.  Then click `Save` again in the Ops Manager web UI.
+
++
+.Google Cloud Platform Config
+|===
+|Field|Value
+
+|Project ID
+|{project_id}
+|===
 
 === Director Config
 
-See http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#director-config[instructions section^].
+. Browse to the `Director Config` section (left margin).
+
++
+[.thumb]
+.Director Config
+image::director-config-gcp.png[Director Config]
+
+. As per the  http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#director-config[documentation^], enter the following values for the specified fields, and accept defaults for all other fields.  Then click `Save`.
+
++
+.Director Config
+|===
+|Field|Value
+
+|NTP
+|169.254.169.254
```

Comment:
just to be devil's advocate.. it makes it harder when they have to check two sets of instructions.  there's potential confusion:  not knowing which set to follow, why there are two, etc.. i feel that ours should primarily guide the reader through the official docs.